react/features/chat/components/PrivateMessageButton.js

```diff
@@ -94,7 +96,11 @@ export function _mapDispatchToProps(dispatch: Function): $Shape<Props> {
  */
 export function _mapStateToProps(state: Object, ownProps: Props): $Shape<Props> {
     return {
-        _participant: getParticipantById(state, ownProps.participantID)
+        _participant: getParticipantById(state, ownProps.participantID),
+        visible:
+            ownProps.visible === undefined
+                ? typeof interfaceConfig === 'undefined' || !interfaceConfig.DISABLE_PRIVATE_MESSAGES
+                : ownProps.visible
     };
```

Comment:
Do you already do a check if the chat button is enabled? So that the private messages button does not display if chat is disabled?

Reply:
Okay, if I'm reading code correct there isn't so maybe it'd be okay to squeeze that change in here too?

Also, there's not an interfaceConfig white that maybe is worth adding the new flag to? The whitelist allows overriding of the interfaceConfig value via a hashparam, which is how the iframe api allows customization of the meeting experience.

Reply:
Yep, added (I hope you meant that)

Reply:
Did you also add to the interfaceConfig whitelist? I don't see the change pushed up.